CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# The @akka-admin-team owns any files in the `/.github/` 
+# folder in the root of the repository and any of its subdirectories.
+.github/settings.yml       @akka-admin-team
```

Comment:
isn't that more specific than the comment suggests? Does a team name need to be prefixed by the org name?

Reply:
Meh, code comments! 🤦🏼‍♂️